drivers/hmis/app/graphql/types/hmis_schema/unit_group.rb

```diff
@@ -0,0 +1,35 @@
+###
+# Copyright 2016 - 2025 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+module Types
+  class HmisSchema::UnitGroup < Types::BaseObject
```

Comment:
maybe include some documentation on the intent of the class
```suggestion
  # Represents a logical grouping of Hmis::Units within a specific Hmis::Hud::Project.
  #   - Supports Coordinated Entry (CE) Configuration:
  #     - Associates a specific `workflow_template` (Hmis::WorkflowDefinition::Template)
  #       to be used for CE Opportunities created for units within this group.
  #     - Enables the definition of CE `eligibility_requirements` and `priority_scheme`
  #       rules (Hmis::Ce::Match::Rule) that apply to all units in the group
  class HmisSchema::UnitGroup < Types::BaseObject
```

Reply:
Thanks. Adding this to the model class instead of the type class